libraries/AP_OpticalFlow/OpticalFlow.cpp

```diff
@@ -59,7 +59,8 @@ const AP_Param::GroupInfo OpticalFlow::var_info[] = {
 
 // default constructor
 OpticalFlow::OpticalFlow(AP_AHRS_NavEKF &ahrs)
-    : _last_update_ms(0)
+    : _last_update_ms(0),
+      _ahrs(ahrs)
```

Comment:
comma in front was a coding style to a place I worked at in the past. It increases clarity during review.